fastlane/lib/fastlane/helper/gradle_helper.rb

```diff
@@ -36,10 +36,14 @@ def task_available?(task)
 
       private
 
+      def escaped_gradle_path
```

Comment:
does it make sense to apply `shellescape` to `gradle_path` in `initialize` so that there is no ambiguity about which path to use? `gradle_path` is still accessible and could be misused.